.circleci/config.yml

```
@@ -27,7 +27,7 @@ defaults: &defaults
 
 version: 2
 jobs:
-  test_build:
+  'build size':
```

We also test the build here.

```suggestion
  'build and size':
```